python/py-tenacity/Portfile

```diff
@@ -17,9 +17,9 @@ homepage            https://github.com/jd/tenacity
 master_sites        pypi:t/tenacity
 distname            tenacity-${version}
```

Comment:
the `master_sites` and `distname` field should be redundant here since these correspond to the defaults set by the python PG

Reply:
You're right, done.